circuits/cpp/src/aztec3/circuits/kernel/private/c_bind.cpp

```diff
@@ -111,6 +114,7 @@ WASM_EXPORT uint8_t* private_kernel__sim(uint8_t const* signed_tx_request_buf,
     return composer.alloc_and_serialize_first_failure();
 }
 
+// jeanmon: only support inner variant
```

Comment:
Why support only inner variant? I know we don't actually generate proofs in our e2e tests but was curious about this comment.

Reply:
This is because the circuit version only have one variant which is the inner one. So far, we splitted only the native PKC into two parts.